github_actions/deploy_cluster/action.yml

```diff
@@ -21,11 +21,11 @@ inputs:
   rke_version:
     description: 'Version of rke to use'
     required: false
-    default: "v1.5.2"
+    default: "v1.6.2"
```

Comment:
Why are you using version 1.6.2 instead of 1.6.3, which is the latest version?

Reply:
Here the k8s supported bu rke 1.6.2: 
- v1.27.16-rancher1-1
- v1.28.14-rancher1-1
- v1.29.9-rancher1-1
- v1.30.5-rancher1-1 (default) 
I can also update update k8s to a more recent version if we want

Reply:
Ok, so stick to this one !